app/components/elements/tooltip_component.rb

```diff
@@ -4,8 +4,7 @@ module Elements
   # Component for rendering a tooltip.
   class TooltipComponent < ApplicationComponent
     def initialize(tooltip: nil)
-      # The tooltip can include a trailing \n that isn't rendered in the HTML but does make for ugly source markup
-      @tooltip = tooltip&.chomp
```

Comment:
We no longer need to chomp off the trailing newline because I removed the trailing newlines. How? Better YAML usage. Instead of:

```yaml
key: >
  lots of text here
```

use this:

```yaml
key: >-
  lots of text here
```